source/tests/GoodWalkerTest.kt

```diff
@@ -0,0 +1,13 @@
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+
+class GoodWalkerTest {
+    @Test
+    fun `Should return string with exact same speed and name of walker specified in GoodWalker class`() {
+        val speed = 10
+
+        val actual = GoodWalker(speed)
```

Comment:
Actual must be final. But u'r use in the assert actual.walk
It's bad practice
Move calling method to actual initialisation